<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBindingsPage</class>
 <widget class="QWidget" name="KeyBindingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>259</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_reset">
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name_filter">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>name filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeySequenceEdit" name="le_sequence_filter">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_reset_filter">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="omm::PreferencesTreeView" name="treeView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>omm::PreferencesTreeView</class>
   <extends>QTreeView</extends>
   <header>preferences/preferencestreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
